Added extra/avidemux

jgmdev opened the pull request:
Changes to upstream:

- Changed build flags as shared here:
   https://avidemux.org/smif/index.php/topic,18851.msg87324.html
- Disabled cpu type check rule on a cmake file
- Added makedepends a52dec and libmad

kmihelich commented:
Why are these changes needed?  32-bit ARM appears to build fine now as-is, and an upstream patch fixes AArch64.

jgmdev commented:
I took the official archlinux x86_64 PKGBUILD which is at version 2.7.4 and tried to build it on aarch64 but it failed complaining about not supported CPU architecture so I removed that CPU check, also the default bootstrap configure flags didn't worked and build failed so the flags needed to also be modified as found on the shared url. Finally since the build is linking against system liba52 and libmad I added those as makedepends. 

I haven't tested avidemux 2.7.5 which hasn't been promoted officially yet.
